tests/Browser/CartTest.php

```diff
@@ -10,8 +10,9 @@ class CartTest extends DuskTestCase
     public function testAddSimpleProduct()
     {
         $this->browse(function (Browser $browser) {
-            $browser->visit($this->testProduct->url)
-                    ->waitUntilIdle()
+            $browser->visit($this->testProduct->url);
+            $browser->script("localStorage['cookie-notice'] = true;");
```

Comment:
Can't we do this by default somewhere else? So we don't have to add this in every test? And add a new test without it so we can test the popup?